.buildkite/scripts/packaging/package-arm.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+IMG_POSTFIX="-SNAPSHOT"
```

Comment:
@oakrizan do we need all of those env vars?

In theory everything should be already exported and a simple `mage package` would be enough